setup.R

```diff
@@ -1,6 +1,8 @@
+#!/bin/env Rscript
```

Comment:
We don't need this as it is not meant to be executed as a standalone script, just embedded within the Rmd file.